x/stakeibc/types/message_add_validators_test.go

```diff
@@ -0,0 +1,93 @@
+package types_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/Stride-Labs/stride/v7/app/apptesting"
+	"github.com/Stride-Labs/stride/v7/x/stakeibc/types"
+)
+
+func TestMsgAddValidators_ValidateBasic(t *testing.T) {
+	validNonAdminAddress, invalidAddress := apptesting.GenerateTestAddrs()
+	adminAddress, ok := apptesting.GetAdminAddress()
+	require.True(t, ok)
+
+	valName1 := "val1"
+	valName2 := "val2"
+	valAddress1 := "cosmosvaloper1"
```

Comment:
should we test with a valid and an invalid address?

Reply:
the invalid address is in one of the test cases (empty address)

See my comment above about validating the true validity of the address